src/gui/optionsdlg.ui

```diff
@@ -2622,12 +2622,22 @@
                  </property>
                  <item>
                   <property name="text">
-                   <string>Pause them</string>
+                   <string>Pause Torrents</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
-                   <string>Remove them</string>
+                   <string>Remove Torrents</string>
+                  </property>
+                 </item>
+                 <item>
+                  <property name="text">
+                   <string>Remove Torrents + Files</string>
+                  </property>
+                 </item>
+                 <item>
+                  <property name="text">
+                   <string>Remove Torrents + Files + Force</string>
```

Comment:
* Suggestion: how about using "Remove torrents + files (force)"?
* Also I prefer only capitalize the first word.

Reply:
+1
A pity the option to strip root folder exists (just for this)
`Remove torrents + folder` would be perfect.

Reply:
>`Remove torrents + folder` would be perfect.

I agree, it is more understandable. But it still requires an explanation for the user that it removes also unrelated files.